server/test/views/questiontypes/ApplicantQuestionRendererFactoryTest.java

```diff
@@ -43,7 +46,10 @@ public void rendererExistsForAllTypes(QuestionType type) throws UnsupportedQuest
 
     ApplicantQuestionRendererFactory factory =
         new ApplicantQuestionRendererFactory(
-            new ApplicantFileUploadRenderer(new AwsFileUploadViewStrategy(), applicantRoutes));
+            new ApplicantFileUploadRenderer(
+                new AwsFileUploadViewStrategy(),
+                applicantRoutes,
+                new SettingsManifest(ConfigFactory.parseMap(ImmutableMap.of()))));
```

Comment:
**suggestion:** can you do `instanceOf(SettingsManifest.class)`? Here and below?

Reply:
`instanceOf` is only defined in `ResetPostgres` and this test class currently doesn't extend `ResetPostgres`. I can make it extend `ResetPostgres` if that's preferred, but I figured if it didn't have to then the test might be faster.

Reply:
Ah got it! There was recently this [PR](https://github.com/civiform/civiform/pull/6694) related to adding ResetPostgres to more tests - I wonder if @gwendolyngoetz has thoughts here.

Reply:
They were expecting a database schema to exist. My database at the time was completely empty. Running the test would then fail since nothing elsewhere had already created the schema. In `bin/sbt-test` run `testOnly package.name.to.YourTest` to run the specific test. You might not see any problems if your unit test database already exists though since the postgres volume is persistent.

Reply:
Ah got it, thanks for the clarification! I did a `docker system prune && docker volume prune` then ran the test and it still passed so I don't think it needs `ResetPostgres` but happy to add it if we want that to be the test standard.

Reply:
If you still have created containers `docker ps -a` pruning wouldn't have removed existing volumes.

Reply:
Oh I deleted the containers first, I should've mentioned that above.